linux_os/guide/system/software/integrity/software-integrity/aide/aide_periodic_cron_checking/rule.yml

```diff
@@ -80,11 +102,26 @@ ocil: |-
     <pre>05 4 * * * root {{{ aide_bin_path }}} {{{ config_command_line }}}--check</pre>
 
     NOTE: The usage of special cron times, such as @daily or @weekly, is acceptable.
+{{%- endif %}}
 
 fixtext: |-
+    Configure the file integrity tool to run automatically on the system at least weekly and to
+    notify designated personnel if baseline configurations are changed in an unauthorized manner.
+    {{%- if "rhel" in product %}}
+    The AIDE tool can be configured to email designated personnel with the use of the cron system.
+    The following example output is generic.
+    It will set cron to run AIDE daily and to send email at the completion of the analysis.
+
+    <pre>
+    $ sudo more /etc/cron.daily/aide
+    #!/bin/bash
+    {{{ aide_bin_path }}} --check | /bin/mail -s "$HOSTNAME - Daily aide integrity check run" root@sysname.mil
```

Comment:
What will we do when now we will have misalignment between the text in rule.yml and bash remediation and OVAL checks? Will we create issues for making them aligned again?

Reply:
IDK, is it more like

1. (RHEL... Must Shut Down The Information System, Restart The Information System), And/**Or** Notify The System Administrator When ..., or is it more like
2. RHEL... Must Shut Down The Information System, (Restart The Information System, And/Or Notify The System Administrator) When ...

The case 1 would mean that nothing has to change, while the case 2 would mean that the notification is optional, and the shutdown, restart and so on have to be carried out.

@Mab879 any thoughts on this?

Reply:
@Mab879 Hi!

Reply:
We should fall back to the more secure option, but that will be on a case-by-case basis.